commands/cluster-shards.md

```diff
@@ -0,0 +1,150 @@
+`CLUSTER SHARDS` returns details about the shards of the cluster.
+A shard is defined as a collection of nodes that serve the same set of slots and that replicate from each other.
+A shard may only have a single master at a given time, but may have multiple replicas.
+It is possible for a shard to not be serving any slots while still having replicas.
+
+The command is suitable to be used by Redis Cluster client libraries in order to understand the topology of the cluster.
+A client should issue this command on startup in order to retrieve the map associating cluster *hash slots* with actual node information.
+This map should be used to direct commands to the node that is likely serving the slot associated with a given command.
+In the event the command is sent to the wrong node, this command can then be used to update the topology of the cluster.
+
+The command returns an array of shards, with each shard containing two fields, 'slots' and 'nodes'. 
```

Comment:
A shard is a map (implicitly understood). See next comment.

Reply:
I didn't follow this comment.

Reply:
"each shard containing two fields, 'slots' and 'nodes'" -- we never stated that it's a map, so that's implicit. In RESP2 it's an array with 4 elements. Like the other comment about RESP2 and RESP3, we don't need to clarify this IIUC.